### PR TITLE
Add dynatrace workflow to the main repo

### PR DESCRIPTION
**Pull request by ahpook**
The website builder only knows about workflows
that live in this repository.

### CODE REVIEW

**Comment by ahpook** [comment]
Local verification:

Screenshot 2020-07-15 15 29 20

File: dynatrace-respond-to-problem/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+"ProblemID":"{ProblemID}",
+"ProblemTitle":"{ProblemTitle}",
+"ProblemURL":"{ProblemURL}",
+"PID" : "{PID}",
+"ProblemSeverity" : "{ProblemSeverity}",
+"ProblemImpact" : "{ProblemImpact}",
+"ProblemDetailsText" : "{ProblemDetailsText}",
+"Tags" : "{Tags}",
+"ProblemDetailsJSON" : {ProblemDetailsJSON}
+}
```
Comment (vine77) [inline dynatrace-respond-to-problem/README.md]:
```suggestion
{
  "State": "{State}",
  "ProblemID": "{ProblemID}",
  "ProblemTitle": "{ProblemTitle}",
  "ProblemURL": "{ProblemURL}",
  "PID": "{PID}",
  "ProblemSeverity": "{ProblemSeverity}",
  "ProblemImpact": "{ProblemImpact}",
  "ProblemDetailsText": "{ProblemDetailsText}",
  "Tags": "{Tags}",
  "ProblemDetailsJSON": "{ProblemDetailsJSON}"
}
```

Comment (vine77) [reply dynatrace-respond-to-problem/README.md]: Should `{ProblemDetailsJSON}` be in quotes like the others to make this code block valid JSON? Not sure how much that matters.

Comment (ahpook) [reply dynatrace-respond-to-problem/README.md]: Good catch, pushed a fix.

**Review by vine77 — APPROVED** [review]
LGTM!

**Pull request merged**